packages/core/src/models/Report.ts

```diff
@@ -116,6 +116,7 @@ export function serializeReport(jsonReport: any): SerializedReport {
     latestPackages: {},
   };
 
+  // eslint-disable-next-line no-prototype-builtins
```

Comment:
Instead of disabling this, can we try and use `Object.prototype.hasOwnProperty.call(foo, "bar");` instead and see how TS likes that.

Reply:
updated and fixes eslint violation.